diff --git a/chrome/android/java/src/org/chromium/chrome/browser/suggestions/SuggestionsDependencyFactory.java b/chrome/android/java/src/org/chromium/chrome/browser/suggestions/SuggestionsDependencyFactory.java
index affa65d69b2d2..81e0b0c5e35d6 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/suggestions/SuggestionsDependencyFactory.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/suggestions/SuggestionsDependencyFactory.java
@@ -37,7 +37,8 @@ public class SuggestionsDependencyFactory {
     }
 
     public MostVisitedSites createMostVisitedSites(Profile profile) {
-        return new MostVisitedSitesBridge(profile);
+        return new org.chromium.chrome.browser.suggestions.mostvisited.BraveMostVisitedSites(
+                profile);
     }
 
     public LargeIconBridge createLargeIconBridge(Profile profile) {
