diff --git a/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/TileGroup.java b/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/TileGroup.java
index bbb3ec132e787..93e66d9470fc7 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/TileGroup.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/TileGroup.java
@@ -306,7 +306,7 @@ public class TileGroup implements MostVisitedSites.Observer {
             new TileSetupDelegate() {
                 @Override
                 public TileInteractionDelegate createInteractionDelegate(Tile tile, View view) {
-                    return new TileInteractionDelegateImpl(
+                    return new BraveTileInteractionDelegateImpl(
                             mContextMenuManager,
                             mTileGroupDelegate,
                             mTileDragDelegate,
